hera_sim/interpolators.py

```diff
@@ -68,6 +88,12 @@ def __init__(self, datafile, pol='xx', **interp_kwargs):
     def __call__(self, lsts, freqs):
         return self._interpolator(lsts, freqs)
 
+    # XXX is there a cleaner way of doing this? (I want to be able to find out
+    # XXX what this interpolator is used for without making an instance of it)
```

Comment:
If you want exactly this behaviour, you're better off just defining `_names` as a class attribute:

```
class Tsky:
    _names = ("Tsky_mdl", )
    ...
```

Reply:
but that is _not_ good encapsulation